doc/install.md

```diff
@@ -24,7 +24,11 @@ You can also build the executables from scratch.
 
 &ensp; 4\. You need to install `ffmpeg` as a dependency.  Run `./install_ffmpeg.sh`.  This will install the dependencies in `~/compiled`.  You need to have `pkg-config` installed.
 
-&ensp; 5\. You can now run `PKG_CONFIG_PATH=~/compiled/lib/pkgconfig go build ./cmd/livepeer/livepeer.go` from the project root directory. To get latest version, `git pull` from the project root directory.
+&ensp; 5\. You can now run `make HIGHEST_CHAIN_TAG=mainnet PKG_CONFIG_PATH=~/compiled/lib/pkgconfig` from the project root directory. To get latest version, `git pull` from the project root directory.
```

Comment:
Could we use a placeholder value here for `HIGHEST_CHAIN_TAG` instead of providing a default tag value to run with especially since the node actually isn't compatible with mainnet right now?

Ex.

"You can now run `HIGHEST_CHAIN_TAG=<CHAIN_NAME> PKG_CONFIG_PATH=~/compiled/lib/pkgconfig make`. See the note below about valid values for `<CHAIN_NAME>`."